src/server/utils/logging/fastifyPlugin.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import url from 'url';
+
+import logger from './logger';
+
+/*
+TIMERS
+`$RequestFullDuration` measures how long it took the whole request to be resolved
+`$RequestOverhead` measures how long it took from when the request was made
+                   until the route handler was called
+`$RouteHandler` measures how long the route handler took to provide an output
+`$ResponseBuilder` measures how long it took for fastify to build the response
+                   after the payload/output from route handled was provided
+
+Why symbols?
+It prevents conflicts with existing keys, native or from other plugins
+*/
+export const $RequestFullDuration = Symbol('$RequestFullDuration');
+export const $RequestOverhead = Symbol('$RequestOverhead');
+export const $RouteHandler = Symbol('$RouteHandler');
+export const $ResponseBuilder = Symbol('$ResponseBuilder');
+
+const TIMERS = {};
+const TENANT_UTILS = {
+  configureRequestLog: ({ log }) => log,
+};
+
+const getLocale = (req) => {
+  // TODO: Verify if `store` is available
```

Comment:
this only affects the locale in logger

Reply:
this is not longer a blocker, the Metrics Server does not inject the store, only the SSR Server which is being migrated in a separate PR